src/renderer/components/SetupTab/index.jsx

```diff
@@ -206,6 +214,24 @@ export default class SetupTab extends React.Component {
     writeParametersToFile(payload);
   }
 
+  async loadParametersFromFile(filepath) {
+    const datastack = await fetchDatastackFromFile(filepath);
+
+    if (datastack.module_name === this.props.pyModuleName) {
+      this.batchUpdateArgs(datastack.args);
+    } else {
+      alert(
+        `Parameter/Log file for ${datastack.module_name} does not match this model: ${this.props.pyModuleName}`);

```

Comment:
This alert message could be more user-friendly. For example, I got this alert:  _"Parameter/Log file for natcap.invest.crop_production_percentile does not match this model: natcap.invest.coastal_vulnerability"_ 
- Could we use the human-friendly model names here?
- I think we usually call it a "datastack file" and not a "parameter file". 
- I would take out the "`${this.props.pyModuleName}`" part. It may be useful for debugging but doesn't really help the user.
- If it's not too much work, it would be nice to say either "Datastack file" or "Log file" as appropriate for the type of file that was dropped.

Reply:
This is minor, but the alert window shows up at the top of the app window. It would be nice to center it over the app.

Reply:
Thanks for the suggestions, here's the new text:

`Datastack/Logfile for Coastal Vulnerability model does not match this model.`

> This is minor, but the alert window shows up at the top of the app window. It would be nice to center it over the app.

Hmm it is centered on Windows. I guess the positioning is controlled by the OS? We may decide to switch away from `window.alert` and design our own at some point.